identity/app/views/profile/accountDetailsForm.scala.html

```diff
@@ -44,6 +45,15 @@ <h2 class="form__heading">Email & password</h2>
                 @inputField(Input(accountDetailsForm("primaryEmailAddress"), ('_label, "Email address"),
                     Symbol("data-test-id") -> "email-address"))
 
+                <li class="form-field form-field--email-change">
+                    @if(changedEmail.isDefined) {
+                        @fragments.inlineSvg("envelope", "icon")
+                        <p class="form__note">
+                            To verify your new email address <b>@changedEmail.get</b> please check your inbox - the confirmation email is on its way. In the meantime you should keep using your old credentials to sign in.<br />
```

Comment:
There is no need to call `.get` on `@changedEmail.get`. Twirl does that for you: https://www.playframework.com/documentation/2.6.x/ScalaTemplates#Displaying-Scala-types

Reply:
what about using `.map()` to avoid getting the option altogether?

Reply:
Nice thanks both